sdk/resourcemanager/Azure.ResourceManager/src/autorest.md

```diff
@@ -593,6 +590,73 @@ directive:
     where: $.definitions.ProviderExtendedLocation.properties.location
     transform: >
       $["x-ms-format"] = "azure-location"
+  # the resource link resource is not using the regular pattern of resources, instead, it is using `/{linkId}` as its creation path
```

Comment:
This directive is so big, how about just custom the code?
Is resource link a spark feature that be widely used? If not can suppress it now, and ask service team to fix it from source?

Reply:
That would be the best solution.
We will write too much custom code which I do not like it this way